faq.md

```diff
@@ -2,7 +2,7 @@
 layout: pagecentered
 title: "FAQ"
 subtitle: "Frequently Asked Questions"
-description: "Frequently Asked Questions | Learn more about Iridium Browser and its features such as updating the browser, synching bookmarks, search engines, extensions/addons, etc."
+description: "Frequently Asked Questions: Learn more about Iridium Browser and it's features such as updating the browser, synching bookmarks, default search engine, audio/video player, extensions and add ons, etc."
```

Comment:
its, nicht it's. Manchmal hat man das Gefühl, es werden Dinge einfach wahllose geändert. :-(

Reply:
🥇